Patch to add Solaris specific paths for mod-alias.  This patch has not
yet been submitted upstream.

--- puppetlabs-apache-1.4.0/manifests/mod/alias.pp.orig	2015-06-03 13:49:26.561897609 -0600
+++ puppetlabs-apache-1.4.0/manifests/mod/alias.pp	2015-06-03 13:49:37.216355218 -0600
@@ -12,6 +12,7 @@ class apache::mod::alias(
     },
     'freebsd' => '/usr/local/www/apache24/icons',
     'gentoo'  => '/usr/share/apache2/icons',
+    'solaris'  => '/var/apache2/2.4/icons',
   }
   apache::mod { 'alias': }
   # Template uses $icons_path
